src/server/NetworkingServer.cpp

```diff
@@ -142,9 +142,14 @@ void NetworkingServer::HandleApiCommand(
     sf::Uint32 api_command_type;
     switch (status) {
         case sf::Socket::Done:
-            if (command_packet >> api_command_type &&
-                api_command_type == (sf::Uint32)APICommandType::register_client) {
+            if (command_packet >> api_command_type) {
+              if (api_command_type == (sf::Uint32)APICommandType::register_client) {
                 RegisterClient(client);
+              }
+
+              if (api_command_type == (sf::Uint32)APICommandType::group) {
+                SetGroupable(client);
```

Comment:
should be indented more. not sure why it's capitalized ...

Reply:
fix plox